content/cli/workspaces.md

```diff
@@ -183,21 +183,22 @@ The top-level properties are as follows:
-| `deleteOutDir`      | boolean             | If `true`, whenever the compiler is invoked, it will first remove the compilation output directory (as configured in `tsconfig.json`, where the default is `./dist`).                                                                    |
-| `assets`            | array               | Enables automatically distributing non-TypeScript assets whenever a compilation step begins (asset distribution does **not** happen on incremental compiles in `--watch` mode).  See below for details.                                  |
+| Property Name       | Property Value Type | Description                                                                                                                                                                                                                           |
+| ------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `webpack`           | boolean             | If `true`, use [webpack compiler](https://webpack.js.org/). If `false` or not present, use `tsc`. In monorepo mode, the default is `true` (use webpack), in standard mode, the default is `false` (use `tsc`). See below for details. |
+| `tsConfigPath`      | string              | (**monorepo only**) Points at the file containing the `tsconfig.json` settings that will be used when `nest build` or `nest start` is called without a `project` option (e.g., when the default project is built or started).         |
+| `webpackConfigPath` | string              | Points at a webpack options file. If not specified, Nest looks for the file `webpack.config.js`. See below for more details.                                                                                                          |
+| `deleteOutDir`      | boolean             | If `true`, whenever the compiler is invoked, it will first remove the compilation output directory (as configured in `tsconfig.json`, where the default is `./dist`).                                                                 |
+| `assets`            | array               | Enables automatically distributing non-TypeScript assets whenever a compilation step begins (asset distribution does **not** happen on incremental compiles in `--watch` mode). See below for details.                                |
+| `watchAssets`       | boolean             | If `true`, watch every non-TypeScript assets. (For more control, see [Assets](cli/monorepo#assets) section below)                                                                                                                     |
```

Comment:
Suggestion:
If `true`, watch every non-TypeScript assets. (For more control, see [Assets](cli/monorepo#assets) section below)
->
If `true`, run in watch-mode, watching specified non-TypeScript assets. (see [Assets](cli/monorepo#assets) section below for more details).

Reply:
If `true`, run in watch-mode, watching **specified** non-TypeScript assets. (see  [Assets](cli/monorepo#assets) section below for more details).
->
If `true`, run in watch-mode, watching **all** non-TypeScript assets. (see  [Assets](cli/monorepo#assets) section below for more details).

It'll watch every assets that define here, even some asserts object is set `watchAssets` to `false`.
What is your opinion?
```
{
  ...
  "compilerOptions": {
    "assets": [
       "**/*.sql", 
       "**/*.html",
       { 
         "include": "**/*.graphql", 
         "exclude": "**/omitted.graphql", 
         "watchAssets": false <-- will not effected if using both, it will take the value from the parent
       },
    ],
    "watchAssets": true <-- if set true, even *.graphql is being watched
  }
}
```

Reply:
already updated. using (all) keywords for now.

Reply:
How about we say something like this:

If true, run in watch-mode, watching **all** non-TypeScript assets. (For more fine-grained control of the assets to watch, see  [Assets](cli/monorepo#assets) section below).   

(Also, see my next comment below)

Reply:
That's looks good to me, already updated.